string_regexp_datatime_task_3.js

```diff
@@ -0,0 +1,7 @@
+// Get the day, month and year of the current date and output it to the console separately
+
+let date = new Date();
+
+console.log("The Year: " + date.getFullYear());
+console.log("The Month: " + date.getMonth());
+console.log("The Day: " + date.getDay());
```

Comment:
function getDay() returns in JS  the day of the week starting from zero. You should have returned the date, for example, today is January 22nd. You should have 22. In your case, this will return today Sunday 6.